### PR TITLE
Bug 2128572: Added application namepace on connected application modal

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
After Fix:
image

File: packages/mco/components/disaster-recovery/drpolicy-list-page/application-status.tsx [hunk 1]
```diff
@@ -113,21 +114,18 @@ const subsFilter = (
 const fetchApplications = (
   subscriptionMapping: SubcsriptionMapping,
   app: ApplicationKind
-): string[] => {
-  let appNames: string[] = [];
-  const appName = app?.metadata?.name;
-  const appNamespace = app?.metadata?.namespace;
+): ApplicationKind[] => {
+  let apps = new Set<ApplicationKind>();
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/drpolicy-list-page/application-status.tsx:118]: I don't think storing `ApplicationKind` in a set will do anything, set will check by reference only and each object reference will be different (even though contents are same)... am I missing something ?

Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/drpolicy-list-page/application-status.tsx:118]: why not only add names of the Application ?

Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/drpolicy-list-page/application-status.tsx:118]: why do we need set ?
application name and namespace combination will be unique anyway...

Comment (GowthamShanmugam) [reply packages/mco/components/disaster-recovery/drpolicy-list-page/application-status.tsx:118]: we dont need a set here, and we dont need a list also here, if one subscription is matching then we can consider the app

**Pull request merged**